src/todo.js

```diff
@@ -0,0 +1,7 @@
+function Todo(title) {
+  this.id = new Date().toLocaleString();
```

Comment:
- [ ] Please adhere to the project, requirements, as I can see that your `todo.id` is using the `date and time` as the unique Id, which is violating the project's actual requirement. At the beginning of the project, you used the numbers as the to-do ID. Kindly use the numbers as ID for your application.

### Requirement
![Screenshot from 2023-08-20 00-44-44](https://github.com/Obote/todo/assets/66476329/8d12660a-a28c-448e-aa98-4985e6537f0d)